pkg/storage/SQL/nda_agreement/get_by_user_id.sql

```diff
@@ -1,7 +1,7 @@
 SELECT
     id,
     user_id,
-    agreed,
+    v2_agreed,
```

Comment:
like above, I'd recommend returning all rows, and having the go model closely match the nda agreement.